src/storage/mem_table_snapshot.cc

```diff
@@ -526,6 +532,9 @@ int MemTableSnapshot::RemoveDeletedKey(const ::openmldb::api::LogEntry& entry, c
         std::set<int> deleted_pos_set;
         for (int pos = 0; pos < entry.dimensions_size(); pos++) {
             std::string combined_key = entry.dimensions(pos).key() + "|" + std::to_string(entry.dimensions(pos).idx());
+            if (entry.has_ts()) {
```

Comment:
if there is `ts_col` in this index, it need to decode ts value from `LogEntry.value()`